freshmaker/pyxis.py

```diff
@@ -49,6 +52,39 @@ def __init__(self, server_url):
         self._server_url = server_url
         # add api version to root url
         self._api_root = urllib.parse.urljoin(self._server_url, "v1/")
+        self._setup_logging()
+
+    def _setup_logging(self) -> None:
+        """ Creates the logging structure of the Pyxis module.
+        
+        We might want a logging level in this module that is different (and probaly higher) than the
+        main-level. Therefore we create a logger that has a custom log level and; otherwise, it will
+        just forward the logs to the main-level logger to handle.
+
+        Also, because of the large amount of data that can be fetched from Pyxis, we will log it
+        into a separate file.
+        """
+        self.logger = logging.getLogger("freshmaker.pyxis")
```

Comment:
The pretty large amount of logs come from `gql` module, not `freshmaker.pyxis`, and these logs are in level `info`, not `debug`, we can change the log level of `gql` module to warning: `internal-repo-url/freshmaker-playbooks/-/blob/master/roles/freshmaker-openshift/templates/fedmsg_logging.py#L43`

We don't have much logs from `freshmaker.pyxis`, with `gql` module logging level changed to `warning`, we probably need to add some logging in `freshmaker.pyxis` for troubleshooting purpose, but we can do that later.

Reply:
Oh, I see... I really misunderstood this issue 😅 
I'll correct it and revert these changes in the pyxis modules.

And just for the curiosity, the circular import didn't happen. That's because I passed the configuration dict as a parameter in the calls to the logger module that are made in `__init__.py`.

In an earlier attempt, I tried to import the configuration dict directly in the logger module, and that resulted indeed a circular error!